packages/dto/src/tables/users/user.rs

```diff
@@ -94,6 +94,14 @@ pub struct User {
     #[serde(default)]
     pub referral_code: String,
 
+    #[api_model(version = v0.9, unique)]
+    #[serde(default)]
+    pub phone_number: Option<String>,
+
+    #[api_model(read_action = find_by_phone_number, skip)]
```

Comment:
Please don't use read_action.
Instead of it, `GET /v2/users?phone-number={}` @victor138128

Reply:
I modified it to remove read_action and implement `/v2/users?phone-number={}` to utilize it.